spec/uploaders/avatar_uploader_spec.rb

```diff
@@ -0,0 +1,75 @@
+# require 'rails_helper'
```

Comment:
Why is this spec commented out in particular? This is the functionality that we want to make sure works correctly?